src/sth_configuration.js

```diff
@@ -0,0 +1,38 @@
+/* globals module, process */
+
+(function() {
+  "use strict";
+
+  var ENV = process.env;
+
+  var dbUsername = ENV.DB_USERNAME || '';
+  var dbPassword = ENV.DB_PASSWORD || '';
+
+  module.exports = {
+    DB_NAME: ENV.DB_NAME || 'test',
+    DB_USERNAME: dbUsername,
+    DB_PASSWORD: dbPassword,
+    DB_AUTHENTICATION: (dbUsername && dbPassword) ?
```

Comment:
Aaand YES!!! And I am reading your comments the Spanish way from top to bottom... :p Good catch, BTW!